lxc/network.go

```diff
@@ -705,6 +709,73 @@ func (c *cmdNetworkGet) Run(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+// Info
+type cmdNetworkInfo struct {
+	global  *cmdGlobal
+	network *cmdNetwork
+}
+
+func (c *cmdNetworkInfo) Command() *cobra.Command {
+	cmd := &cobra.Command{}
+	cmd.Use = i18n.G("info [<remote>:]<network>")
+	cmd.Short = i18n.G("Get runtime information of networks")
+	cmd.Long = cli.FormatSection(i18n.G("Description"), i18n.G(
+		`Get runtime information on networks`))
```

Comment:
It writes `on` here but `of` right before. :)

Reply:
Oops, fixing